packages/react-native-web/src/vendor/react-native/Animated/nodes/AnimatedInterpolation/__tests__/index-test.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-env jasmine, jest */
+
+import AnimatedInterpolation from '..';
+import AnimatedValue from '../../AnimatedValue';
+
+describe('AnimatedInterpolation', () => {
```

Comment:
Please move this test to `.../Animated/__tests__/`. I'd rather not restructure the vendored code too

Reply:
Moved this test, and copied interpolation tests from react native. Could put them in Animated and refactor them to use Animated.Value instead if needed.